docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -7,54 +7,20 @@ description: How to configure Teleport Database Access with self-hosted MySQL
 
 ## Create Certificate/Key Pair
 
-Teleport uses mutual TLS for authentication to MySQL instances. As such,
-self-hosted MySQL instances must be configured with Teleport's certificate
-authority and a certificate/key pair that Teleport can validate.
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
 
-To create these secrets, use `tctl auth sign` command. It can be run either
-[locally](../../architecture/overview.mdx#tctl) on the auth server, or from the
-client machine after logging in with `tsh login` (for example, when using
-[Teleport Cloud](../../cloud/introduction.mdx)).
-
-In the latter case, user must be allowed to impersonate the system role `Db` in
-order to be able to generate the database certificate, by having the following
-allow rule in their role:
-
-```yaml
-allow:
-  impersonate:
-    users: ["Db"]
-    roles: ["Db"]
-```
-
-Run the command:
+Create the secrets:
 
 ```sh
 # Export Teleport's certificate authority and generate certificate/key pair
-# for host db.example.com with a 3-month validity period.
-tctl auth sign --format=db --host=db.example.com --out=server --ttl=2190h
+# for host db.example.com with a 1-year validity period.
+$ tctl auth sign --format=db --host=db.example.com --out=server --ttl=2190h
```

Comment:
`2190` is 3-months.

Per our conversation, I'd think 3-moths, 6-months, or 12-months would be appropriate with a preference for 3-months or 6-months.